src/MSIdentityScaffolding/Microsoft.DotNet.MSIdentity/CodeReaderWriter/CodeModifierConfigs/cm_dotnet_blazorserver.json

```diff
@@ -131,16 +131,9 @@
                 "NumberOfSpaces": 4
               }
             },
-            {
-              "Block": "app.UseAuthentication()",
-              "InsertAfter": "app.UseRouting()",
-              "LeadingTrivia": {
-                "Newline": true
-              }
-            },
             {
               "Block": "app.UseAuthorization()",
-              "InsertAfter": "app.UseAuthentication()",
```

Comment:
This configuration looks like it is adding the authorization services explicitly under certain options:

https://github.com/dotnet/Scaffolding/blob/057e3fd14c5dd433ed6070e543ee97533ac5e38e/src/MSIdentityScaffolding/Microsoft.DotNet.MSIdentity/CodeReaderWriter/CodeModifierConfigs/cm_dotnet_blazorserver.json#L18-L22

So we can remove `UseAuthorization` in those cases as well. Not sure how hard that is.